packages/apps/web-viewer-test/src/components/home/ViewerHome.tsx

```diff
@@ -157,16 +153,16 @@ const ViewerHome: React.FC = () => {
           }),
           new MeasureToolsUiItemsProvider(),
         ]}
-        extensions={[
-          new LocalExtensionProvider({
-            manifestPromise: LocalExtension.manifestPromise,
-            main: LocalExtension.main,
-          }),
-          new RemoteExtensionProvider({
-            jsUrl: "http://localhost:3001/dist/index.js",
-            manifestUrl: "http://localhost:3001/package.json",
-          }),
-        ]}
+        // extensions={[
```

Comment:
Intended to leave commented-out?

Reply:
yea, this is the test app, and these extensions dont really work anymore (we dont support tools now, just basic vp stuff)